crates/ruff/src/checkers/ast/analyze/deferred_for_loops.rs

```diff
@@ -16,7 +16,7 @@ pub(crate) fn deferred_for_loops(checker: &mut Checker) {
             })
             | Stmt::AsyncFor(ast::StmtAsyncFor {
                 target, iter, body, ..
-            }) = &checker.semantic.stmt()
+            }) = &checker.semantic.current_statement()
```

Comment:
Shouldn't this be `statement_current` for consistency (I prefer `current_statement` but it seems inconsistent)

Reply:
I strongly prefer `current_statement`.